### ISSUE TITLE
All projectile commands are failing on Develop

### ISSUE THREAD

**Issue by hackeryarn** [hlissner/doom-emacs#792]
### Observed behavior

All `projectile` commands fail with `Wrong type argument: arrayp, nil`.

This is a fresh install of doom from `develop`. The only extra modules enabled are `magit`, `javascripts`, and `ivy` in favor of `helm`. 

### Expected behavior

Projectile commands to work.

### Steps to reproduce

`SPC SPC`

All other projectile commands seem to fail as well.

### System information

<details>
<summary>Click to expand</summary>

```
- OS: darwin (x86_64-apple-darwin16.7.0)
- Emacs: 26.1 (May 28, 2018)
- Doom: 2.0.9 (develop 3131bda98c45098d6a41880a26478ee4e6bf396f)
- Graphic display: t (daemon: nil)
- System features: JPEG RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS LCMS2
- Details:
  ```elisp
  elc count: 0
  uname -a:  Darwin stlwm158495.accounts.root.corp 16.7.0 Darwin Kernel Version 16.7.0: Thu Jun 21 20:07:39 PDT 2018; root:xnu-3789.73.14~1/RELEASE_X86_64 x86_64
  modules:   (:feature (evil +everywhere) file-templates (lookup +devdocs +docsets) snippets spellcheck (syntax-checker +childframe) workspaces :completion (ivy +fuzzy) :ui doom-dashboard doom-modeline doom-quit evil-goggles hl-todo nav-flash treemacs (popup +all +defaults) vc-gutter vi-tilde-fringe window-select :editor rotate-text :emacs ediff electric hideshow imenu vc :tools magit :lang emacs-lisp javascript markdown (org +attach +babel +capture +export +present) (sh +fish) :config)
  packages:  n/a
  exec-path: (~/.rvm/gems/ruby-2.4.1/bin/ ~/.rvm/gems/ruby-2.4.1@global/bin/ ~/.rvm/rubies/ruby-2.4.1/bin/ ~/.cargo/bin/ /usr/local/bin/ /usr/bin/ /bin/ /usr/sbin/ /sbin/ /usr/local/sbin/ ~/bin/ ~/.nvm/versions/node/v8.11.0/bin/ ~/.cargo/bin/ ~/.rvm/gems/ruby-2.4.1/bin/ ~/.rvm/gems/ruby-2.4.1@global/bin/ ~/.rvm/rubies/ruby-2.4.1/bin/ ~/.rvm/bin/ ~/go/bin/ ~/.fzf/bin/ /usr/local/Cellar/emacs-plus/26.1/libexec/emacs/26.1/x86_64-apple-darwin16.7.0/)
  ```
```
</details>

**Comment by hlissner** [comment]
This appears to be an issue with the counsel-projectile package. I'll apply a quick fix for this later tonight if it hasn't already been resolved.

**Comment by hlissner** [comment]
87c8f6e should fix the problem for now, but it should be reversed when the issue is fixed upstream. I'll leave this issue open until it has been resolved upstream.

**Comment by hackeryarn** [comment]
Thank you! This fixed the issue for me.

**Comment by hackeryarn** [comment]
@hlissner I spoke too soon. Most of the command work. However, `Switch to project`(`SPC p p`) still throws the same error, and `def-project-mode!` inside `lang/javascript` doesn't seem to identify the project properly. These issues look to be related.

**Comment by orther** [comment]
Just tested https://github.com/ericdanan/counsel-projectile/pull/92 and it fixed `Switch to project (SPC p p)` now too.

**Comment by hackeryarn** [comment]
Thanks for the update @orther I look forward to the fix being available!

**Comment by hlissner** [comment]
As of ddf107a5, counsel-projectile is temporary disabled. This should resolve the issue (and the base projectile commands will still work), but at the cost of counsel-projectile-specific functionality.

**Comment by hlissner** [comment]
It appears this issue was resolved upstream, so I've reversed ddf107a. 

Please update counsel-projectile. Feel free to reopen this if the issue isn't fully resolved.
